méli-mélo/th-ai-answers/style.css

```diff
@@ -0,0 +1,52 @@
+/* AI answers components styles */
+.aia-banner {
+	background: #1a2838;
+	color: #fff;
+	position: relative;
+}
+
+.aia-banner p {
+	font-size: 1.125rem;
+	margin: 0;
+	padding: .75rem 0 .75rem 1rem;
+}
+
+/*.aia-banner .aia-close {
```

Comment:
Can you add a textual comment that explain those commented CSS code, or can you remove it for now until we have a working example for it?